runFlask.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
+LOG_DIR="$VAG_COMMON_DIR/log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR=$HOME/FlaskApp
```

Comment:
what's the purpose of this Dir? I don't see any files being put into it or used.